wfe/web-front-end.go

```diff
@@ -624,13 +625,30 @@ func (wfe *WebFrontEndImpl) RevokeCertificate(response http.ResponseWriter, requ
 	}
 }
 
+func (wfe *WebFrontEndImpl) logCsr(remoteAddr string, cr core.CertificateRequest, registration core.Registration) {
+	var csrLog = struct {
+		RemoteAddr   string
+		CsrBase64    string
+		Registration core.Registration
+	}{
+		RemoteAddr:   remoteAddr,
+		CsrBase64:    base64.StdEncoding.EncodeToString(cr.Bytes),
```

Comment:
If you make CsrBase64 a []byte, `json.Marshal` will do the base64 encoding for you. I'm sorry I didn't see this earlier.